README.md

```diff
@@ -1,6 +1,50 @@
 [![Community Project header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Project.png)](https://opensource.newrelic.com/oss-category/#community-project)
 
-# docs-website
+# New Relic Docs
+
+This repo contains the documentation website code and Markdown source files for  our preview Docs site: docs-preview.newrelic.com. When it's ready, all of our docs will live here.
+
+Read on to learn more about who we are and how you can contribute.
+
+## We'd like your help
+
+From early on, our Docs site has been managed by a team of writers. We've always welcomed contributions from anyone at New Relic. We credit our technical accuracy and comprehensive documentation to this openness. In a sense, it's documentation as conversation.
+
+Open sourcing our docs seemed like a great way to expand this conversation beyond New Relic.
+
+We're here working every day to improve our docs, but we'd love to hear from you. Come join the conversation.
+
+## Get started
+
+On every one of our docs, you can [create an issue]([link to issue template page](https://github.com/newrelic/docs-website/issues/new/choose)) or [edit a page]([link to more detailed instructions about contributing](https://github.com/newrelic/docs-website/blob/develop/CONTRIBUTING.md)). 
+
+### Create an issue
+
+Issues are a quick way to give us feedback about our docs. We'll review your issue and follow up with you if we have any questions.
+
```

Comment:
I would add a second link/CTA to file an Issue here. Maybe as its own line so it really stands out. Otherwise someone could F-scan for "issue" and then not see a link here.